src/Tests/dotnet.Tests/CommandTests/ToolRunCommandTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.CommandFactory;
+using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.DotNet.ToolManifest;
+using Microsoft.DotNet.ToolPackage;
+using Microsoft.DotNet.Tools.Tool.Run;
+using Microsoft.Extensions.DependencyModel.Tests;
+using Microsoft.Extensions.EnvironmentAbstractions;
+using NuGet.Frameworks;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.Tests.Commands.Tool
+{
+    public class ToolRunCommandTests 
+    {
+        private readonly IFileSystem _fileSystem;
+        private const string ManifestFilename = "dotnet-tools.json";
+        private readonly string _testDirectoryRoot;
+        private DirectoryPath _nugetGlobalPackagesFolder;
+        private readonly LocalToolsResolverCache _localToolsResolverCache;
+
+        public ToolRunCommandTests() 
+        {
+            _fileSystem = new FileSystemMockBuilder().UseCurrentSystemTemporaryDirectory().Build();
+            _nugetGlobalPackagesFolder = new DirectoryPath(NuGetGlobalPackagesFolder.GetLocation());
+            string temporaryDirectory = _fileSystem.Directory.CreateTemporaryDirectory().DirectoryPath;
+            _localToolsResolverCache = new LocalToolsResolverCache(
+                _fileSystem,
+                new DirectoryPath(Path.Combine(temporaryDirectory, "cache")));
+            _testDirectoryRoot = _fileSystem.Directory.CreateTemporaryDirectory().DirectoryPath;
+
+        }
+
+        [Fact]
+        public void WhenRunWithRollForwardOptionItShouldIncludeRollForwardInNativeHost()
+        {
+            var parseResult = Parser.Instance.Parse($"dotnet tool run $TOOLCOMMAND$ --roll-forward Major");
+
+            var toolRunCommand = new ToolRunCommand(
+            parseResult);
```

Comment:
Nit: This doesn't seem like it needs to be two lines.  For statements that do go over one line, usually the lines after the first one should be indented for better readability.